spec/amp-email-format.md

```diff
@@ -0,0 +1,379 @@
+| `<amp-list>` [[Example](https://ampbyexample.com/components/amp-list/)] | Remotely fetches JSON data that will be rendered by an [`<amp-mustache>`](https://www.ampproject.org/docs/reference/components/amp-mustache).<br><br>**Note:** Binding to the `[src]` attribute is not allowed. Including user credentials with `credentials="include"` is also prohibited. |
+| `<template type="amp-mustache">` [[Example](https://ampbyexample.com/components/amp-mustache/)] | A Mustache template markup to render the results of an `amp-list` call. |
+
+### Layout
+
+| Element | Description |
+| ------- | ----------- |
+| `<amp-accordion>` [[Example](https://ampbyexample.com/components/amp-accordion/)] | A UI element that facilitates showing/hiding different sections. |
+| `<amp-carousel>` [[Example](https://ampbyexample.com/components/amp-carousel/)] | A carousel UI component. |
+| `<amp-sidebar>` [[Example](https://ampbyexample.com/components/amp-sidebar/)] | A sidebar for navigational purposes. |
+| `<amp-image-lightbox>` [[Example](https://ampbyexample.com/components/amp-image-lightbox/)] | A lightbox for containing images. |
+| `<amp-lightbox>` [[Example](https://ampbyexample.com/components/amp-lightbox/)] | A lightbox for containing content. |
+| `<amp-fit-text>` [[Example](https://ampbyexample.com/components/amp-fit-text/)] | A helper component for fitting text within a certain area. |
+| `<amp-timeago>` [[Example](https://ampbyexample.com/components/amp-timeago/)] | Provides a convenient way of rendering timestamps. |
+
+### Media
+
+| Element | Description |
+| ------- | ----------- |
+| `<amp-img>` [[Example](https://ampbyexample.com/components/amp-img/)] | An AMP component that replaces `<img>`.<br><br>**Note:** Binding to `[src]` is not allowed. |
+| `<amp-anim>` [[Example](https://ampbyexample.com/components/amp-anim/)] | Embeds GIF files.<br><br>**Note:** Binding to `[src]` is not allowed. |
+
+## CSS requirements
+
+### Specifying CSS in an AMP document
+
+All CSS in any AMP document must be included in a `<style amp-custom>` tag within the header. Inline style attributes are not allowed in AMP.
+
+```html
+...
+<style amp-custom>
+  /* any custom styles go here. */
+  body {
+    background-color: white;
+  }
+  amp-img {
+    border: 5px solid black;
+  }
+  amp-img.grey-placeholder {
+    background-color: grey;
+  }
+</style>
+...
+</head>
+```
+
+**Note:** The entire `<style>` tag cannot exceed 50,000 bytes. The validator will check for this.
+
+### Allowed CSS properties and selectors
+
+CSS allowed within email messages vary depending on the email provider. For  reference, the list of CSS properties and values allowed in Gmail can be found at [Gmail Supported CSS Properties & Media Queries](https://developers.google.com/gmail/design/reference/supported_css).
+
+## Document dimensions
+
+* **Optimal width**: 800px or less (any wider and things will get cut off).
```

Comment:
Curious: Should the "cutting off" behavior be email client-specific or is this part of the recommendation we include in the spec?

Reply:
I still think it is important to mention it, it applies to all clients.

Reply:
Reworded this to mention client-specific behavior.